StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
```

Comment:
How does people know what's the available options here? only NL? maybe UK? maybe we should set all available values here such as `Given(@"I am on the (NL|UK|AMS) website")`, or maybe you should properly raise handle unexpected code in `setCountryCode(String countryCode)` with the function

Reply:
What do you mean `website`... I suggest it should be `home page`